packages/uniforms/src/bridges/SimpleSchema2Bridge.js

```diff
@@ -31,8 +31,7 @@ try {
     );
 } catch (_) { /* Ignore it. */ }
 
-// export default class SimpleSchema2Bridge extends Bridge {
-export class SimpleSchema2Bridge extends Bridge {
+export default class SimpleSchema2Bridge extends Bridge {
```

Comment:
Yeah, that's a terrible typo.

_(publish before going to bed is generally a bad idea)_